app/models/report.rb

```diff
@@ -19,6 +19,9 @@
 #  action_taken_at            :datetime
 #  rule_ids                   :bigint(8)        is an Array
 #  application_id             :bigint(8)
+#  forwarded_at               :datetime
+#  forwarded_to_domains       :string           default([]), not null, is an Array
+#  forwarded_by_id            :bigint(8)
```

Comment:
The idea is that these will supersede the existing `forwarded` boolean; For received Flag activities, we probably want to set `forwarded_at` and `forwarded_by` to when we received the Flag, and the actor for that activity.